docs/sphinx/user_guide/cook_book/reduction.rst

```diff
@@ -91,3 +91,20 @@ Here a simple sum reduction is performed using RAJA::
 The results of these operations will yield the following values:
 
  * vsum.get() == 1000
+
+
+Another option for the execution policy when using the cuda or hip backends are
+the base policies which have a boolean parameter to choose between the general
+use ``cuda/hip_exec`` policy and the ``cuda/hip_exec_with_reduce`` policy.::
+
+  // static constexpr bool with_reducers = ...;
+  // using exec_policy = RAJA::cuda_exec_base<with_reducers, 256>;
+  // using exec_policy = RAJA::hip_exec_base<with_reducers, 256>;
+
+Another option for the reduction policy when using the cuda or hip backends are
+the base policies which have a boolean parameter to choose between the atomic
+``cuda/hip_reduce_atomic`` policy and the non-atomic ``cuda/hip_reduce`` policy.::
+
+  // static constexpr bool maybe_atomic = ...;
```

Comment:
For consistency with the discussion above, would it be better to change the name of the boolean variable `maybe_atomic` --> `with_atomic`?

Reply:
Ya let's do that.